src/Common/CommonServices.ts

```diff
@@ -472,14 +481,66 @@ export interface IGlobalMessageBanner {
      * Links to supply to the format arguments in `messageFormat`
      */
     messageLinks?: IGlobalMessageLink[];
+
+    /**
+     * Where the banner should appear.
+     */
+    position?: string;
```

Comment:
These were intentionally left out of the interface for third-parties. Position in particular, but also customIcon (we would need/want to update it to allow them to specify their own image url, not an icon name) and dismissable (I think we want all third-party banners to be dismissable).

Reply:
I figured as much but wasn't sure when it was added. I'll cut them